test/data/openapi2/petstore-expanded.json

```
@@ -184,6 +184,17 @@
           }
         }
       }
+    },
+    "/pets/cat": {
+      "get": {
+        "description": "Returns pets which are cats",
+        "operationId": "find pets which are cats",
+        "responses": {
+          "200": {
+            "description": "empty schema"
```

imo) 204 No Content I think the response body is empty as a specification. So why not make the test data follow the common use case?

```suggestion
        "responses": {
          "204": {
            "description": "The resource was deleted successfully."
```

Refs: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/204#compatibility_notes

---

In my recent experiences, I've noticed the following scenarios:

1. status_code=200 && empty response
=> This PR addresses this case. While it's technically correct to use status_code=204, there are instances where APIs are operated with 200, and ideally, I'd like to accommodate this case as well.

2. status_code=204 && empty response
=> While it's being skipped [here](https://github.com/interagent/committee/blob/10a4ebc14566ad8de02380cf68412b5b68e67367/lib/committee/middleware/response_validation.rb#L39) for 204, it seemed like a better approach to modify it to skip within Committee::SchemaValidator::HyperSchema::ResponseValidator. If it's not an inconvenience, I'm considering addressing this in a separate PR. Could you please share your thoughts on this?

---

Thank you for your detailed explanation!
I understood the intent and thought it make sense. So I agree with your opinion.